app/code/community/Bolt/Boltpay/controllers/OrderController.php

```diff
@@ -72,13 +72,50 @@ public function saveAction()
 
             if ($order->isEmpty()) {
                 $session_quote = $checkout_session->getQuote();
-                $boltHelper->createOrder($reference, $session_quote->getId());
+                $order = $boltHelper->createOrder($reference, $session_quote->getId());
+
+                if ($order->isEmpty()) {
+                    // Generic message is passed if no exceptions were thrown and error still wasn't created
+                    Mage::throwException( "We are unable to process your order at this time.  Please try again later." );
+                }
             }
-            /////////////////////////////////////////////////////////
+
+            $this->getResponse()
+                ->setHeader('Content-type', 'application/json', true)
+                ->setBody(
+                    Mage::helper('core')
+                        ->jsonEncode(
+                            array(
+                                'result' => 'success'
+                            )
+                        )
+                )
+            ;
 
         } catch (Exception $e) {
             Mage::helper('boltpay/bugsnag')->notifyException($e);
-            throw $e;
+
+            /////////////////////////////////////////////////////////
+            /// Since the order failed, we must cancel the Bolt Order
```

Comment:
Please please don't do this. In case of inventory missing, invicta team needs to check items and may do partial capture.